manifest.json

```diff
@@ -16,15 +16,15 @@
     },
```

Comment:
Same here, can't suggest in the correct spot
```
      "address": "13RKFFy5qmibyaS6N3Uw18UBDwGhhsXpk6jDn7SMenfDrQ6o6eh",
      "signature": "yAYjiYx5I3IIfCsWTUlZaR95OBy2634S2uoLtFzijKc+SEbomNUo1x6QL2nfflAhxnfV3pt7llcWqUjtIGPsCA=="
```

Reply:
@6CornerDefender and this one